lib/matplotlib/widgets.py

```diff
@@ -453,10 +454,12 @@ def __init__(self, ax, label, valmin, valmax, *, valinit=0.5, valfmt=None,
         self.val = valinit
         self.valinit = valinit
 
-        defaults = {'facecolor': 'white', 'edgecolor': '.75', 'size': 10}
+        defaults = {'marker': 'o', 'facecolor': 'white',
+                    'edgecolor': '.75', 'size': 10}
         handle_style = {} if handle_style is None else handle_style
         marker_props = {
-            f'marker{k}': v for k, v in {**defaults, **handle_style}.items()
+            f'{"marker" * (not k.startswith("marker")) + k}': v \
```

Comment:
Your suggestion would broaden the API so that we'd accept `handle_style={'markerfacecolor': ...}`. It would be a separate debate whether that is reasonable, and it would also create some ambiguity `handle_style={'markerfacecolor': 'r', 'facecolor': 'g'}`. So for now, I suggest to only expand to accept "marker":

```suggestion
            f'{"marker" * (k != marker) + k}': v
```
Also this arithmethic is quite clever and we might want to be more explcit

```
            def prefix_marker(key):
                return key if key == "marker" else "marker" + key
                
            makrer_props = {
                prefix_marker(k): v for ...
```

Reply:
Thank you for your comments, they are very insightful and so to the point. I absolutely agree with everything you said :)
Would you mind pointing out to where this 'prefix_marker' function should be defined?

Reply:
It can be a nested function unless we have some other place to use it, and it'd be more for self-documentation.